routes.py

```diff
@@ -0,0 +1,12 @@
+from flask import Flask
```

Comment:
What is flask?

Reply:
It's a python library that lets you spin up web servers.